proposals/permissions_behavior_on_update.md

```diff
@@ -1,14 +1,16 @@
-# Proposal: Suppress permissions warning on update
+# Proposal: Allow to change permissions behavior on extension's update
 
 ** Proposal Summary**
 
-Add a new boolean property to the manifest that changes the default behavior of added "permissions" during the extension's update. Instead of showing a "warning," permissions shouldn't be granted; they should behave like they are optional until an extension is requested. It doesn't change behavior during install time or for permission that doesn't trigger a warning.
+Add a new property, "permissions_behavior_on_update," to the manifest to alter the newly added "permissions" behavior during the extension's update. When set to "defer," newly added permissions on update won't be granted by default; they will behave like optional permissions and must be explicitly requested by the extension. Also, the default browser behavior will NOT be applied (warning, turn off, stop further updates, etc.).
+
+The new behavior isn't applicable during install time or for permission that doesn't trigger a warning.
 
 Extension manifest file example:
 
 ```json
 {
-  "suppress_permissions_warning_on_update": true
+  "permissions_behavior_on_update": "defer"
```

Comment:
Probably disregard; however want to mention it, we could be further future-proofed by making it an object.
```json
{
  "permissions_behavior": {
    "updates": "defer",
  }
}
```

allowing other types of permission behaviours we might want to introduce in the future.